netsim/cli/connect.py

```diff
@@ -163,17 +163,18 @@ def create_command_list(host: Box, args: argparse.Namespace, rest: list) -> list
   else:
     return rest
 
-def connect_to_node(
+def connect_to_node(      
       args: argparse.Namespace,
       rest: list,
       topology: Box,
+      node: typing.Optional[str] = None, 
       log_level: LogLevel = LogLevel.INFO) -> typing.Union[bool,int,str]:
-  node = args.host
+  
   host_data = outputs_common.adjust_inventory_host(
```

Comment:
This would now crash if the node would not be specified (i.e. it would be set to None)